modules/backup/variables.tf

```diff
@@ -23,3 +23,53 @@ variable "s3_access_logs_bucket_name" {
   description = "Define name for the S3 access logs"
   type        = string
 }
+
+# KMS CMK
+
```

Comment:
As these are currently set, without being exposed on root level they are as good as hardcoded in the cmk.tf module. 
With the exception of `enable_s3_kms_key` and `s3_key_admin_arn`

Reply:
Changed all.